.github/workflows/release.yaml

```diff
@@ -0,0 +1,40 @@
+name: Release Workflow
+on:
+  workflow_dispatch:
+    inputs:
+      release-version:
+        description: 'Version being released'
```

Comment:
If the `v` prefix is a part of the version, I'd mention it in the description, just in case

Reply:
Otherwise LGTM

Reply:
Done. Added